### PR TITLE
A page about settings in pharo

### PR DESCRIPTION
**Pull request by dupriezt**

### CODE REVIEW

File: General/Settings.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+* [The Settings Browser](#the-settings-browser)
+* [Declaring a Setting](#declaring-a-setting)
+	+ [As quick as possible](#as-quick-as-possible)
+	+ [WARNING!: Default value of a setting](#warning---default-value-of-a-setting)
+	+ [Elements of a Setting Declaration](#elements-of-a-setting-declaration)
+		- [Id of the setting (#mySetting)](#id-of-the-setting---mysetting-)
+		- [Is it a setting or a group node? (setting:)](#is-it-a-setting-or-a-group-node---setting--)
+		- [Where is the variable this setting affects? (target:)](#where-is-the-variable-this-setting-affects---target--)
+		- [Label and Description (label: description:)](#label-and-description--label--description--)
+		- [Parent (parent:)](#parent--parent--)
```
Comment (juliendelplanque) [inline General/Settings.md:20]: Those links are not working, I guess because of the extra dashes in the references.

File: General/Settings.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+- Users can change the value of any setting defined in the image with the **Settings Browser** window. No need to build a custom UI for it.
+- It is easy to add settings to an already existing application. A setting is basically just a class variable (or an instance variable of a singleton object) whose value can be set from the Settings Browser.
+- No dependency. Your application does not depend on the **Pharo Settings Framework**, and the Pharo Settings Framework does not depend on your application.
+
+## Table of Contents
+
+* [The Settings Browser](#the-settings-browser)
+* [Declaring a Setting](#declaring-a-setting)
+	+ [As quick as possible](#as-quick-as-possible)
+	+ [WARNING!: Default value of a setting](#warning---default-value-of-a-setting)
```
Comment (juliendelplanque) [inline General/Settings.md:14]: This link is not working, probably because of the extra dashes in the reference

File: General/Settings.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+The Settings Browser is by default able to manage simple preference types. These default possibilities are generally enough. But there are some situations where it can be very helpful to be able to handle more complex preference values.  
+As an example, let us focus on the text selection preferences. We have the primary selection and three other optional kinds of text  selection, the secondary selection, the ‘find and replace’ selection and the selection bar. For all selections, a background color can be set. For the primary, the secondaryand the find and replace selection, a text color can also be chosen.
+
+*More information in section 5.8 "Extending the Settings Browser" of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/).*
+
+## Credits
+Thomas Dupriez.  
+This page is based on chapter 5 of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/): The Settings Framework  
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
```
Comment (juliendelplanque) [inline General/Settings.md:292]: I think you can remove this last line, we never specify the tool we use to generate the toc. :-)

**Comment by dupriezt** [comment]
I fixed the links and removed the credits for the table of contents

**Pull request merged**